main-api.tf

```diff
@@ -85,7 +85,7 @@ module "rds" {
   app_name            = var.app_name
   app_env             = "${local.app_env}-tf"
   engine              = "postgres"
-  engine_version      = "12.7"
+  engine_version      = "12.8"
```

Comment:
AWS automatically updates DB engine versions if you've set the parameter correctly.  It is done during a maintenance window.  I only find out because Terraform complains.  ;-)